Source Code/Ticket Management System/auth_menu.c

```diff
@@ -37,10 +36,8 @@ void setEmail()
 
 void setPassword()
 {
-	wchar_t* context;
 	wchar_t input[100];
-	fgetws(input, 10, stdin);
-	wcstok(input, L"\n", &context);
+	input_wchar(input, 10);
```

Comment:
`input` is of size `100` but `input_wchar` is called with `10`.
I thought I fixed that bug before? Someone must have reverted it accidentally.

Reply:
`input_wchar(&input, 100);`

Reply:
I left it as it was I saw it was 10 so I left it as 10 tried to make as lesser changes as I could to working parts.

Reply:
Has to be `input_wchar(&input, 100);`